src/main/java/uk/gov/hmcts/reform/civil/utils/NocNotificationUtils.java

```diff
@@ -18,51 +18,95 @@ public static String getPreviousSolicitorEmail(CaseData caseData) {
         if (isRespondent2NewSolicitor(caseData)) {
-            otherSolicitor1Email = caseData.getApplicantSolicitor1UserDetails().getEmail();
-            otherSolicitor1OrgId = caseData.getApplicant1OrganisationPolicy().getOrganisation().getOrganisationID();
-        } else if (caseData.getChangeOfRepresentation().getCaseRole().equals(CaseRole.APPLICANTSOLICITORONE)) {
-            otherSolicitor1Email = caseData.getRespondentSolicitor2EmailAddress();
-            otherSolicitor1OrgId = caseData.getRespondent2OrganisationPolicy().getOrganisation().getOrganisationID();
-        } else {
-            otherSolicitor1Email = caseData.getRespondentSolicitor1EmailAddress();
-            otherSolicitor1OrgId = caseData.getRespondent1OrganisationPolicy().getOrganisation().getOrganisationID();
+            return RecipientData.builder()
+                .email(caseData.getApplicantSolicitor1UserDetails().getEmail())
+                .orgId(caseData.getApplicant1OrganisationPolicy().getOrganisation().getOrganisationID())
+                .build();
+        } else if (isApplicant1NewSolicitor(caseData)) {
+            if (!isOtherPartyLip(caseData.getRespondent1OrganisationPolicy())) {
+                return RecipientData.builder()
+                    .email(caseData.getRespondentSolicitor1EmailAddress())
+                    .orgId(caseData.getRespondent1OrganisationPolicy().getOrganisation().getOrganisationID())
+                    .build();
+            }
+        } else if (isRespondent1NewSolicitor(caseData)) {
+            return RecipientData.builder()
+                .email(caseData.getApplicantSolicitor1UserDetails().getEmail())
+                .orgId(caseData.getApplicant1OrganisationPolicy().getOrganisation().getOrganisationID())
+                .build();
         }
+        return null;
     }
 
-    private static void setOtherSolicitor2(CaseData caseData) {
+    private static RecipientData getOtherSolicitor2(CaseData caseData) {
         if (isRespondent2NewSolicitor(caseData)) {
-            otherSolicitor2Email = caseData.getRespondentSolicitor1EmailAddress();
-            otherSolicitor2OrgId = caseData.getRespondent1OrganisationPolicy().getOrganisation().getOrganisationID();
+            if (!isOtherPartyLip(caseData.getRespondent1OrganisationPolicy())) {
+                return RecipientData.builder()
+                    .email(caseData.getRespondentSolicitor1EmailAddress())
+                    .orgId(caseData.getRespondent1OrganisationPolicy().getOrganisation().getOrganisationID())
+                    .build();
+            }
         } else {
-            otherSolicitor2Email = caseData.getRespondentSolicitor2EmailAddress();
-            otherSolicitor2OrgId = caseData.getRespondent2OrganisationPolicy().getOrganisation().getOrganisationID();
+            if (getMultiPartyScenario(caseData).equals(TWO_V_ONE)) {
+                return RecipientData.builder()
```

Comment:
hmmm, if it's a 2v1, do we need to notify applicantsolicitor1 again? i think it's worth a check with BAs/Leanne